docs/04-resources/06-guides/powpeg/advanced-operations/index.md

```diff
@@ -2,38 +2,38 @@
 sidebar_position: 1200
 sidebar_label: Advanced Operations 
 title: "Overview | Advanced Operations"
-description: "See how to perform advanced operations on the two way peg app"
+description: "See how to perform advanced operations on the PowPeg App"
 tags: [2 way peg app, powpeg, peg-in, peg-out, bridge, rsk, rootstock]
 ---
 
-This section contains detailed instructions on how to perform advanced operations on the 2 way peg app. 
+This section contains detailed instructions on how to perform advanced operations on the PowPeg app. 
```

Comment:
application name should be: "PowPeg App"

Reply:
Updared. Thanks